src/games/strategy/engine/framework/headlessGameServer/HeadlessServerSetup.java

```diff
@@ -183,4 +183,10 @@ public void postStartGame() {
     final GameData data = m_gameSelectorModel.getGameData();
     data.getProperties().set(PBEMMessagePoster.PBEM_GAME_PROP_NAME, false);
   }
+
+  @Override
+  public boolean isMetaSetupPanelInstance() {
+    return false;
```

Comment:
- I like the explicit override everywhere rather than following an inheritance heirarchy

In both situations it's the wrong answer. The meta-setup panel should not be grouped together with the other Setup panels. A few things are shared, but it's not the right abstraction, the SetupPanel should not have this method at all. For now it's better than having an instanceof check, and is meant to be relatively low effort for the moment (with the real effort being to fix the inheritance hierarchy of the setup panels)